arch/x86_64/mcount.S

```diff
@@ -51,7 +51,7 @@ GLOBAL(mcount)
 	movq %rsp, %rdx
 	.cfi_def_cfa_register rdx
 
-	/* align stack pointer to 16-byte */
+	/* align stack pointer to 16-bit */
 	andq $0xfffffffffffffff0, %rsp
```

Comment:
This is actually to clear last 4 bits in the `rsp` register (stack pointer).  Clearing 4 bits makes the content a multiple of 16 (= 2^4).  So after the instruction the stack pointer will point somewhere in the stack memory aligned by 16 **byte**.

Reply:
Thank you for your comments!